src/index.ts

```diff
@@ -1,32 +1,37 @@
 import express from 'express';
 
+import { logger } from './logger';
 import { GroupController, UserController } from './controllers/index';
 import { UserJoiValidator, GroupJoiValidator } from './validators/index';
 import { GroupService, UserService } from './services/index';
 import { GroupRepository, UserRepository, UserGroupRepository, initDBData } from './data-access/index';
 
+process.on('uncaughtException', (err) => {
```

Comment:
you need to add one more listener:
`Add Unhandled promise rejection listener to log errors.`